.yarnrc

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-lastUpdateCheck 1572004604492
+lastUpdateCheck 1575384140796
```

Comment:
Is there a command you execute to update that file?

Reply:
https://github.com/tbergq/monorepo/blob/master/apps/notes/yarn/yarn.md#yarn-policies-set-version